<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Демонстрационное приложение</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>view/favicon.ico</normaloff>view/favicon.ico</iconset>
  </property>
  <property name="accessibleName">
   <string>r1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QWebView" name="webView">
      <property name="accessibleName">
       <string>myweb</string>
      </property>
      <property name="url">
       <url>
        <string>http://www.google.kz/?gfe_rd=cr&amp;ei=18YyV-DuEof07gTh1JGgAg</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="my_open"/>
    <addaction name="separator"/>
    <addaction name="my_close"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Действие</string>
    </property>
    <addaction name="my_run"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="my_open">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="my_close">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="my_run">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Запуск</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
